README.md

```diff
@@ -20,11 +20,55 @@ Compiled binary releases are available for download from the [releases page](htt
 
 [![Build Status](https://travis-ci.org/GafferHQ/gaffer.svg?branch=master)](https://travis-ci.org/GafferHQ/gaffer)
 
-Gaffer is a fairly large project, and as such has a fairly complex build process. Before you start, make sure you have the following prerequisites installed on your system, which will be used to perform the build itself:
+Gaffer is a fairly large project, and as such has a fairly complex build process. Before you start, make sure you have the following prerequisites installed on your system, which will be used to perform the build itself.
 
-- [SCons](https://www.scons.org)
-- [Inkscape](https://inkscape.org)
-- [Sphinx](https://www.sphinx-doc.org/) 1.4 or higher (optional)
+
+### Build Requirements ###
+
+From time to time, this list may change. For a complete, accurate, and up-to-date method of installing the prerequisites on CentOS, refer to the [Docker setup](https://github.com/GafferHQ/build/blob/master/Dockerfile) we use for building automatic releases.
+
+> **Note:** Specific package names may differ depending on your Linux distribution and repository.
+
+
+#### Main Build Requirements ####
+
+> **Note:** Large Linux distros focused on usability, such as CentOS and Ubuntu, ship with many of these packages by default.
+
+Package Name | Minimum Version
+------------ |:--------------:
+**Build Dependencies** | -
+[gcc](https://gcc.gnu.org/index.html) | 6.3.1
+[cmake](https://cmake.org) | 3
```

Comment:
Not needed to build Gaffer - just used to build dependencies. I think these instructions should assume the use of prepackaged dependencies, so we can omit. There's several more of these actually :

- CMake
- Doxygen
- nasm
- zlib-devel (I think)

Reply:
Could you clarify: do you mean those are all for building the dependencies, and GCC is in, while CMake is out?

Reply:
Yes. The ones I listed (CMake, Doxygen, nasm and zlib-devel) are not needed to build Gaffer, so we should remove them from the list. Looks like you've removed CMake in a later commit, but not the others...